### DISCUSSION TITLE
How do you use the GPUPixelSourceImage class? in android project

### DISCUSSION THREAD

**Discussion by minshoki** [pixpark/gpupixel#81] [Q&A]
I'm trying to use it in my android project.

Is there a sample code that can apply a filter to a bitmap using GPUPixelSourceImage and receive the final result as a bitmap?

**Comment by thiepdientuu** [comment]
Hi

**Reply by qianjujun** [reply]
+1

**Reply by thiepdientuu** [reply]
+2

**Reply by GrossGhost** [reply]
+3
